spec.html

```diff
@@ -28899,7 +28899,7 @@ <h1>parseInt ( _string_, _radix_ )</h1>
           1. Set _R_ to 10.
         1. If _stripPrefix_ is *true*, then
           1. If the length of _S_ is at least 2 and the first two code units of _S_ are either *"0x"* or *"0X"*, then
-            1. Remove the first two code units from _S_.
+            1. Set _S_ to the substring of _S_ from 2.
```

Comment:
Here and elsewhere, mild preference for having "index" when using literal indices instead of an alias named e.g. "index" for the form "substring of _S_ from _n_"

```suggestion
            1. Set _S_ to the substring of _S_ from index 2.
```

Reply:
So the rule is: when using the "substring of" form, where the "to" part is omitted, and the from-index is an integer literal, insert "index" before that literal. Right?

I've added a fixup commit to this effect.

Reply:
> So the rule is: when using the "substring of" form, where the "to" part is omitted, and the from-index is an integer literal, insert "index" before that literal. Right?

Yep, exactly.